### DISCUSSION TITLE
Can I specify the frames used for caiman?

### DISCUSSION THREAD

**Discussion by cleeneuro** [flatironinstitute/CaImAn#1402] [Q&A]
I have a two-photon images of dendrites, stored in a single big tiff file containing approximately 60,000 frames. Periodically, there is significant motion artifact (due to something in the behaviour set up) affecting several hundred frames, making them unusable. When I process the data using CaImAn, the motion is too extreme to correct, and I believe this causes the spatial components to appear blurry or smeared. Is there a simple way to exclude specific frames while still preserving the overall temporal structure of the dataset? 

Thank you!

**Comment by kushalkolar** [comment]
What if you increase max shifts to a ridicuously high number, I've been able to motion correct rigid motion where even a third of the FOV shifts. 

If not, just keep track of the frames indices that are unusable and slice them out? If you want to map the sliced array back to time [`pynapple`](https://github.com/pynapple-org/pynapple) makes this easy by letting you slice in time and not just frame indices.
